lib/RequestSender.js

```diff
@@ -13,7 +13,7 @@ const EventHandlers = {
 
 module.exports = class RequestSender
 {
-  constructor(ua, request, eventHandlers)
+  constructor(ua, request, eventHandlers, config2=null)
```

Comment:
I will not check this file right now, because I may refactor this class so it can receive an arbitrary credential.

Reply:
if JsSIP will support multiple  (user-name, password, realm) it will be good.
It was workaround.